### PR TITLE
README changes for the newer spec format.

### PR DESCRIPTION
**Pull request by mrunalp**
Signed-off-by: Mrunal Patel <mrunalp@gmail.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,72 +36,104 @@ PID   USER     COMMAND
...
 ### OCF Container JSON Format:
 
-Below is a sample `container.json` configuration file. It assumes that
+Below is a sample `config.json` configuration file. It assumes that
 the file-system is found in a directory called `rootfs` and there is a
 user named `daemon` defined within that file-system.
 
 ```json
 {
-    "version": "0.1.1",
```
Comment (crosbymichael) [inline README.md]: why are all these changes here? i'm just using the output from `runc spec` and pasting it in the readme.

Comment (mrunalp) [inline README.md]: Formatter that I used messed it up. Fixed.

**Comment by LK4D4** [comment]
LGTM

**Comment by mrunalp** [comment]
Rebased.

**Comment by alecbenson** [comment]
+1
These instructions worked perfectly for me. Thanks!

**Pull request merged**